### DISCUSSION TITLE
Rodrigues space (RF space)

### DISCUSSION THREAD

**Discussion by sarialzerey** [mtex-toolbox/mtex#2059] [Ask Anything]
Hi, 

Is it possible to generate Rodrigues space RF-space in MTEX?
Please let me know!

Best,
Sari

**Comment by ralfHielscher** [comment]
What is RF space? What do you mean by generate the space in MTEX?

Ralf.

**Reply by sarialzerey** [reply]
It's displaying the Rodrigues vectors on a 'fundamental zone' which has a shape depending on the crystal symmetry. The input of the Rodrigues vector is the Axis/Angle pairs where the axis defines the direction of Rodrigues vector and the angle defines the magnitude of the vector. Rodrigues space describes the orientation and the misorientation. 

https://www.jstor.org/stable/pdf/79938.pdf?refreqid=fastly-default%3A633dbff93411e85d06be9ca8d31d80ce&ab_segments=&origin=&initiator=&acceptTC=1

**Comment by ralfHielscher** [comment]
Have a look at https://mtex-toolbox.github.io/OrientationFundamentalRegion.html and replace `'axisAngle`' by `'rodrigues'` in the plotting commands.

Ralf.
